ui/app/js/controllers/disable_updates_modal.js

```diff
@@ -0,0 +1,33 @@
+angular.module('app').controller('DisableUpdatesModalCtrl',
+  function($scope, $modalInstance, EmergencyShutoffs, product, channel){
+    $scope.product = product;
+    $scope.channel = channel;
+    $scope.saving = false;
+
+    $scope.disableUpdates = function() {
+      $scope.saving = true;
+      EmergencyShutoffs.create($scope.product, $scope.channel)
+        .success(function(response) {
+          $modalInstance.close(response);
+          sweetAlert("Disabling Updates", "Updates was disabled successfully!", "success");
+        })
+        .error(function(response, status) {
+          console.log(response);
+          if (typeof response === 'object') {
+            sweetAlert("Form submission error", response.data, "error");
+          }
+        })
+        .finally(function() {
+          $scope.saving = false;
+        });
+    };
+
+    $scope.cancel = function() {
+      $modalInstance.dismiss('cancel');
+    };
+
+    (function() {
+      var audio = new Audio('audio/emergency.mp3');
```

Comment:
I know the bug specifically suggested this, but now that I've experienced it....I'm going to have to take it back. I think this will be very annoying, hehe.

Reply:
I know, but I could not ignore that request to try freak out the user!